explainer.md

```diff
@@ -215,6 +243,162 @@ third-party then signs either the exchanges in a package or the package as a
 whole using a certificate whose metadata reflects whichever property was
 reviewed for.
 
+## Loading sketch
+
+When an **embedder** prefetches or embeds an enveloped signed exchange, or a
+client navigates from the embedder to an enveloped signed exchange, the
+client goes through several steps to open the envelope and load the signed
+resource.
+
+### Fetch the physical URL
+
+The client won't know that a URL holds a signed exchange until it receives the
+`Content-Type` in the response, so the initial request is identical to any other
+request in the same context. It follows redirects, is constrained by the
+embedder's and any parent frame's Content Security Policy, and goes through the
+embedder's or the physical URL's Service Worker.
+
+Once the response comes back, its `Content-Type: application/signed-exchange`
+header tells the client that it represents a signed exchange, but it's initially
+treated like any other response: the Response object shown to the Service Worker
+is the bytes of the `application/signed-exchange` resource, not the response
+inside it, and the client follows the remaining steps only if the Service Worker
```

Comment:
'only if the Service Worker responds with...' part felt a bit confusing (in this paragraph it's not clear if we're talking about fetching the physical URL in general or the case that with the embedder's SW).